Modules/RosterManager/RosterConfiguration.lua

```diff
@@ -169,18 +161,54 @@ local TRANSFORMS = {
 }
 
 function RosterConfiguration:inflate(data)
-    for i, key in ipairs(self:fields()) do
+    for i, key in ipairs(fields()) do
         self._[key] = TRANSFORMS[key](data[i])
     end
 end
 
-function RosterConfiguration:deflate()
```

Comment:
oh you are going to hate me Samm 

(๑•́ㅿ•̀๑)ᔆᵒʳʳᵞ

`deflate()`
is used in RaidManager LedgerEntries

Reply:
Haha no worries; will get back to it when i can ;-)